### ISSUE TITLE
KTH results 

### ISSUE THREAD

**Issue by JingJerry** [Yijunmaverick/FlowGrounded-VideoPrediction#4]
I follow the instructions to test the KTH dataset  by using your pre-train model and the results are as below. The synthesized frame is a little bit weird and looks like flow. Is there anything wrong with my implementation?

![test_val](https://user-images.githubusercontent.com/25581086/59431717-1ee20900-8e18-11e9-82e2-494be21b7bb0.png)

**Comment by ardasnck** [comment]
@JingJerry I was wondering are these results from original Torch implementation or from your own implementation in different libraries? 
In any case; is there any chance that you can inform the learning rate, batch size and all the other hyperparameters that you used for **flow prediction part**? 
Thanks.

**Comment by Katreenlooper** [comment]
Hey @JingJerry ,
Can you please tell me how to run the code and test it on my own raw video?
I put my own raw video in the following (datasets/DTexture/raw), but what should i change in the script(collectinfo.py and others) to make it work?
I appreciate your answer..
Thanks
